atom/browser/native_browser_view_views.h

```diff
@@ -16,9 +16,7 @@ class NativeBrowserViewViews : public NativeBrowserView {
   ~NativeBrowserViewViews() override;
 
   uint8_t GetAutoResizeFlags() { return auto_resize_flags_; }
-  void SetAutoResizeFlags(uint8_t flags) override {
-    auto_resize_flags_ = flags;
-  }
+  void SetAutoResizeFlags(uint8_t flags) override;
```

Comment:
not a blocker, but what was the issue with not inlining this? size-would-be-larger warning?

Reply:
Virtual methods are never actually inlined: https://www.chromium.org/developers/coding-style/chromium-style-checker-errors#TOC-Virtual-Method-Out-of-lining